src/layer/aeroway.js

```diff
@@ -15,8 +15,8 @@ const iconLayout = {
     "match",
     ["get", "class"],
     "military",
-    "military_airport",
-    "airport",
+    "poi_military_plane",
+    "poi_plane",
```

Comment:
Several entries in the taginfo project file template need to be updated:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/d12a7486607dcf128b6d55d6acc616e6cabecf33/scripts/taginfo_template.json#L132

Reply:
Updated 3 entries for military airports.